includes/api/v2/class-wc-rest-system-status-tools-v2-controller.php

```diff
@@ -530,16 +525,6 @@ public function execute_tool( $tool ) {
 				$message = __( 'Tax rates successfully deleted', 'woocommerce' );
 				break;
 
-			case 'reset_tracking':
-				if ( ! class_exists( 'WC_Tracker' ) ) {
-					include_once WC_ABSPATH . 'includes/class-wc-tracker.php';
-				}
-				WC_Tracker::opt_out_request();
```

Comment:
Looks like this sends a final event when tracking is opted out.  Should we add this under the option update hook or is it being removed?

```php
add_action( 'update_option_ woocommerce_allow_tracking', 'track_opt_out', 10, 2 );

function track_opt_out( $old_value, $value ) {
    if ( 'yes' !== $old_value && 'no' !== $value ) {
        return;
    }

    WC_Tracker::opt_out_request();
}
```

Reply:
This opting-out will be captured in Tracks Settings changes, https://github.com/woocommerce/woocommerce/pull/22810.

Nice catch! I can go ahead and remove `WC_Tracker::opt_out_request`.